pkg/virt-config/featuregate/feature-gates_test.go

```diff
@@ -85,4 +85,13 @@ var _ = Describe("Feature Gate", func() {
 		Expect(featuregate.FeatureGateInfo(fg1.Name)).To(Equal(&fg1clone))
 		Expect(featuregate.FeatureGateInfo(fg2.Name)).To(Equal(&fg2))
 	})
+
+	It("HyperVL1VH gate is registered as Alpha and has spec detector", func() {
```

Comment:
I would suggest removing the test since it's not adding any hyperv feature gate specific tests.